ontology/cdno.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: ontology_detail
+id: cdno
+title: Crop Dietary Nutrition Ontology
+build:
+  checkout: git clone https://github.com/Southern-Cross-Plant-Science/cdno.git
+  system: git
+  path: "."
+contact:
+  email: l.andres.hernandez.18@student.scu.edu.au
+  label: Liliana Andres Hernandez
```

Comment:
Please include your GitHub user name here.

```
contact:
  email: l.andres.hernandez.18@student.scu.edu.au
  label: Liliana Andres Hernandez
  github: LilyAndres
```

Reply:
Ok, sure thing. I added this info and changed the license to 3.0